src/modules/services/submodule/mainnetSwap/submodule2/eth/views/ViewServicesMainnetSwapEthOptions.vue

```diff
@@ -2,7 +2,7 @@
   <div class="flex justify-between text-sm">
     <div><span class="text-gray-400">Swap > ETH > </span> <span class="text-blue-primary font-bold">Select option</span></div>
     <div>
-      <router-link :to="{ name: 'ViewServices'}" class="font-bold" active-class="accounts">All Services</router-link>
+      <router-link :to="{ name: 'ViewServices'}" class="font-bold" active-class="accounts">Home</router-link>
```

Comment:
this also, same question as above

Reply:
@sleepyOwl14 Title has been updated. Thanks